src/modules/fancyzones/editor/FancyZonesEditor/GridZone.xaml.cs

```diff
@@ -23,8 +23,6 @@ public partial class GridZone : UserControl
         private const string GridZoneBackgroundBrushID = "GridZoneBackgroundBrush";
         private const string SecondaryForegroundBrushID = "SecondaryForegroundBrush";
         private const string AccentColorBrushID = "SystemControlBackgroundAccentBrush";
-        private const string CanvasCanvasZoneBorderBrushID = "CanvasCanvasZoneBorderBrush";
```

Comment:
Positive these are dead?

Reply:
Yes. I just double checked:
![image](https://user-images.githubusercontent.com/16760760/154816556-e4bdba28-b71d-4d96-8b98-9b2f914c0843.png)

No occurrences of this string within the folder `src\modules\fancyzones`